Assets/Examples/Test/Editor/Driver/Testforscene1TestSample.cs

```diff
@@ -2147,5 +2147,24 @@ public void TestFindObjectFromObject(By by, string value, string nameOfChild)
             var child = parent.FindObjectFromObject(by, value);
             Assert.True(child.name == nameOfChild);
         }
+
+        [Test]
+        public void TestImplicitTimeout()
+        {
+            var timeStart = DateTime.Now;
+            altDriver.SetImplicitTimeout(5);
+            altDriver.WaitForObject(By.NAME, "Capsule");
+            var timeEnd = DateTime.Now;
+            var time = timeEnd - timeStart;
+            Assert.LessOrEqual(time.TotalSeconds, 5);
+            altDriver.SetImplicitTimeout(20);
+        }
```

Comment:
This test doesn't check what you expect to check. This test should see if the WaitForObject wait for the time you set.